generators/app/templates/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import Login from './layout';
+
+class LoginContainer extends Component {
+  handleSubmit = () => {
+    // TODO implement function
+  };
+
+  handleChange = () => {
```

Comment:
I don't like using the same function to handle all input changes. I'd have a handleEmailChange and handlePasswordChange